candidate-testing.js

```diff
@@ -1,58 +1,87 @@
-const input = require('readline-sync');
+const input = require('readline-sync')
 
-// TODO 2: modify your quiz app to ask 5 questions //
+let candidateName= "";
+let question = "Who was the first American woman in space? ";
+let correctAnswer = "Sally Ride";
+let candidateAnswer = "";
+let questions = ["Who was the first American woman in space? ",
+"True or false: 5 kilometer == 5000 meters? ",
+"(5 + 3)/2 * 10 = ? ",
+"Given the array [8, 'Orbit', 'Trajectory', 45], what entry is at index 2? ",
+"What is the minimum crew size for the ISS? "
+];
+let correctAnswers = [	"Sally Ride","true","40","Trajectory","3"];
+let candidateAnswers = [];
 
-// TODO 1.1a: Define candidateName // 
-let candidateName;
-// TODO 1.2a: Define question, correctAnswer, and candidateAnswer //
-let question;
-let correctAnswer;
-let candidateAnswer;
-let questions;
-let correctAnswers;
-let candidateAnswers;
+function askForName(){
+candidateName = input.question("Please enter your name: ");
+}
 
+function askQuestion(){
+  for(let i=0; i<questions.length; i++){
+    candidateAnswers.push(input.question(questions[i]))
+  }
+}
 
-function askForName() {
-  // TODO 1.1b: Ask for candidate's name //
 
-}
+function gradeQuiz(candidateAnswers){
+
+console.log("Candidate Name: "+candidateName);
+let count=0;
+let grade=0;
+
+for(let i=0; i<questions.length; i++){
 
-function askQuestion() {
-  // TODO 1.2b: Ask candidate the question and assign the response as candidateAnswer //
+  console.log(`Question ${questions[i]}`);
+  console.log(`Your answer ${candidateAnswers[i]}`);
+  console.log(`Correct answer ${correctAnswers[i]}`);
+  console.log();
 
 
+if(candidateAnswers[i].toLowerCase()===correctAnswers[i].toLowerCase()){
+  count++;
+
+}
 }
 
-function gradeQuiz(candidateAnswers) {
+grade = (count/questions.length)*100;
 
-  // TODO 1.2c: Let the candidate know if they have answered the question correctly or incorrectly // 
+if(grade>=80){
```

Comment:
This block could be indented better to make it more readable.   There's a style guide in the course book appendices that you can use as a guide.